website/docs/reference/commands/clone.md

```diff
@@ -49,7 +49,7 @@ You can clone nodes between states in dbt Cloud using the `dbt clone` command. T
 
   - Set up your **Production environment** and have a successful job run.
   - Enable **Defer to production** by toggling the switch in the lower-right corner of the command bar.
-    <Lightbox src="/img/docs/dbt-cloud/defer-toggle.jpg" width="80%" title="Select the 'Defer to production' toggle on the bottom right of the command bar to enable defer in the dbt Cloud IDE."/>
+    <Lightbox src="/img/docs/dbt-cloud/defer-toggle.png" width="200%" title="Select the 'Defer to production' toggle on the bottom right of the command bar to enable defer in the dbt Cloud IDE."/>
```

Comment:
i still see the hover notification and i also recommend tidying up the red box around the defer toggle as it looks a little misaligned. 
![Screenshot 2024-12-09 at 11 16 17](https://github.com/user-attachments/assets/be8071ee-2fbd-4ac1-9e9b-9c2a622ecaae)

also, what is the difference between this new image and the current img?

![Screenshot 2024-12-09 at 11 16 55](https://github.com/user-attachments/assets/b614954c-04dd-421a-91ff-98e6a636e69b)